src/screenshot.ts

```diff
@@ -0,0 +1,46 @@
+import { Page, Base64ScreenShotOptions } from "puppeteer";
```

Comment:
The`Base64ScreenShotOptions` is not used in the file, you can 🔥 it

```suggestion
import { Page } from "puppeteer";
```